### PR TITLE
added adjustMovementSpeed function

### PR DESCRIPTION
**Pull request by plally**

### CODE REVIEW

File: lua/autorun/server/sv_pvp_movespeed.lua [hunk 1]
```diff
@@ -29,11 +29,30 @@ local function playerIsInBuild( ply )
...
+	local wepCount = 0
+	for k, weapon in pairs(weapons) do
+		if nonEffectedWeapons[weapon:GetClass()] == nil then
+            wepCount =  wepCount+1
+            
+        end
+	end
+    multiplier = movementMultiplier(wepCount) 
+    ply:SetRunSpeed(400*multiplier)
+    ply:SetWalkSpeed(200*multiplier)
```
Comment (overlordbleck) [inline lua/autorun/server/sv_pvp_movespeed.lua]: Is it possible to get the default run and walk values from a function

File: lua/autorun/server/sv_pvp_movespeed.lua [hunk 2]
```diff
@@ -29,11 +29,30 @@ local function playerIsInBuild( ply )
...
+    -- calculate movement speed multiplier
+end
+local function adjustMovementSpeed(ply) 
+
+	local weapons = ply:GetWeapons(ply)
+	local wepCount = 0
+	for k, weapon in pairs(weapons) do
+		if nonEffectedWeapons[weapon:GetClass()] == nil then
+            wepCount =  wepCount+1
+
```
Comment (overlordbleck) [inline lua/autorun/server/sv_pvp_movespeed.lua]: I really think we should have a separate function so that the code is neater.

Comment (plally) [reply lua/autorun/server/sv_pvp_movespeed.lua]: as in a separate function for the weapon count?

Comment (overlordbleck) [reply lua/autorun/server/sv_pvp_movespeed.lua]: Yeah

**Comment by overlordbleck** [comment]
Is this working?

**Comment by plally** [comment]
the movementMultiplier function is still just a placeholder, i didnt fix the issue with that math

**Comment by plally** [comment]
the movementMultiplier function could still use some updating, i just made it functional

**Pull request merged**